src/entries/popup/hooks/send/useSendAsset.ts

```diff
@@ -65,14 +64,27 @@ export const useSendAsset = () => {
     [],
   );
 
+  const combinedAssets = useMemo(
+    () =>
+      Array.from(
+        new Map(
+          [...customNetworkAssets, ...assets].map((item) => [
+            item.uniqueId,
+            item,
+          ]),
+        ).values(),
+      ),
+    [assets, customNetworkAssets],
+  );
+
   const allAssets = useMemo(
     () =>
-      uniqBy([...assets, ...customNetworkAssets], 'uniqueId').sort(
+      combinedAssets.sort(
         (a: ParsedUserAsset, b: ParsedUserAsset) =>
           parseFloat(b?.native?.balance?.amount) -
           parseFloat(a?.native?.balance?.amount),
       ),
-    [assets, customNetworkAssets],
+    [combinedAssets],
   );
```

Comment:
Do you know if this would impact hidden and pin functionality ? Since if there are two same `uniqueId`'s and i hide one of them both of them will be hidden since we hide by `uniqueId`. I might be wrong, but can you test that ?

Reply:
that would be the expected right? since we hide uniqueid

Reply:
Uploading Screen Recording 2024-04-17 at 14.14.49.mov…

how do i uhide? lol

Reply:
https://github.com/rainbow-me/browser-extension/assets/12115171/90aa88ed-c84a-4993-9f51-1f6fe32b9eda

Reply:
Ahh i see. Alright then it looks great to me 🙏

Reply:
Opps. Approved 🙏